Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -775,38 +778,38 @@ public static function cacheConfigurationProvider() : array
         return [
             'metadata_cache_default' => [
                 'expectedAliasName' => 'doctrine.orm.default_metadata_cache',
-                'expectedAliasTarget' => 'doctrine.orm.cache.pool.cache.system',
+                'expectedAliasTarget' => 'doctrine.cache.provider.cache.doctrine.orm.default.metadata',
```

Comment:
Not 100% happy with this service name :confused: But we use the pool name in the id and prefix it...

Any better suggestions?